bin/phpunit.sh

```diff
@@ -4,4 +4,21 @@ if [[ $RUN_PHPCS == 1 || $SHOULD_DEPLOY == 1 ]]; then
 	exit
 fi
 
-if [ -f "phpunit.phar" ]; then php phpunit.phar -c phpunit.xml.dist $@; else ./vendor/bin/phpunit $@; fi;
+CURRENT_PHP_MAJOR_VERSION=$(php -r 'echo PHP_MAJOR_VERSION;')
+CURRENT_PHP_MINOR_VERSION=$(php -r 'echo PHP_MINOR_VERSION;')
+
+# The PHPUnit version inside composer.json is not compatible with PHP 7.2 and above
+# Update this constant if you wish to bump supported PHP major version
+SUPPORTED_PHP_MAJOR_VERSION_FOR_PHPUNIT_INSTALLED_VIA_COMPOSER_JSON=7
+SUPPORTED_PHP_MINOR_VERSION_FOR_PHPUNIT_INSTALLED_VIA_COMPOSER_JSON=3
+
+if [ $CURRENT_PHP_MAJOR_VERSION -gt $SUPPORTED_PHP_MAJOR_VERSION_FOR_PHPUNIT_INSTALLED_VIA_COMPOSER_JSON ]; then
+	./vendor/bin/phpunit $@;
+else
+	if [ $CURRENT_PHP_MINOR_VERSION -ge $SUPPORTED_PHP_MINOR_VERSION_FOR_PHPUNIT_INSTALLED_VIA_COMPOSER_JSON ]; then
+		./bin/phpunit6 -c phpunit.xml.dist $@
+	else
+    	chmod +x ./bin/phpunit6
```

Comment:
I may miss something here. This means no tests are executed for PHP < 7.3?

Also, the following runs are only show `Running the tests.` then finish without any test. I think that is due to this command.

https://github.com/Automattic/woocommerce-payments/runs/5350917490?check_suite_focus=true#step:5:168

https://github.com/Automattic/woocommerce-payments/runs/5350917508?check_suite_focus=true#step:5:168

https://github.com/Automattic/woocommerce-payments/runs/5350917927?check_suite_focus=true#step:5:168

Reply:
This is fixed. Thank you for finding this.